app/models/owner.js

```diff
@@ -18,7 +18,7 @@ export default VcsEntity.extend({
   name: attr('string'),
   login: attr('string'),
   channels: attr(),
-  isSyncing: attr('boolean'),
```

Comment:
Is there any reason to remove the type from attribute? Usually it's better to provide types, because it's not only helps to visually understand the data, but also helps to _ensure_ the expectations. If no type is passed to the `attr`, Ember Data will use `raw` transform, which doesn't parse the input and assigns whatever comes from API. Thus, if for some reason the format will change on API, the UI will just brake.